ReportSimulationOutput/measure.rb

```diff
@@ -123,6 +123,11 @@ def arguments(model)
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('user_output_variables', false)
+    arg.setDisplayName('Generate Timeseries Output: User-Defined Output Variables')
+    arg.setDescription('Optionally generates timeseries output variables. If multiple output variables, use a comma-separated list. Do not include key values; by default all key values will be requested.')
```

Comment:
```suggestion
    arg.setDisplayName('Generate Timeseries Output: EnergyPlus Output Variables')
    arg.setDescription('Optionally generates timeseries EnergyPlus output variables. If multiple output variables are desired, use a comma-separated list. Do not include key values; by default all key values will be requested. Example: "Zone People Occupant Count, Zone People Total Heating Energy"')
```

I took these output variable names from your test below so I assume it's correct, but I haven't tested it.